src/hotspot/share/gc/shenandoah/mode/shenandoahMode.hpp

```diff
@@ -32,16 +32,16 @@ class ShenandoahHeuristics;
 #define SHENANDOAH_CHECK_FLAG_SET(name)                                     \
   do {                                                                      \
     if (!(name)) {                                                          \
-      err_msg message("GC mode needs -XX:+" #name " to work correctly");    \
-      vm_exit_during_initialization("Error", message);                      \
+      const char *msg = "GC mode needs -XX:+" #name " to work correctly";   \
+      vm_exit_during_initialization("Error", msg);                          \
     }                                                                       \
   } while (0)
 
 #define SHENANDOAH_CHECK_FLAG_UNSET(name)                                   \
   do {                                                                      \
     if ((name)) {                                                           \
-      err_msg message("GC mode needs -XX:-" #name " to work correctly");    \
-      vm_exit_during_initialization("Error", message);                      \
+      const char* msg = "GC mode needs -XX:-" #name " to work correctly";   \
+      vm_exit_during_initialization("Error", msg);                          \
```

Comment:
Same as above, can be inlined into one call. No need for the temporary variable.

Reply:
Please see my comment in the thread above.

Reply:
Whatever you decide with Alexey is fine. The temporary variable is my least favorite option.